src/main/java/net/snowflake/client/jdbc/SnowflakeResultSetSerializableV1.java

```diff
@@ -552,6 +553,9 @@ public static SnowflakeResultSetSerializableV1 create(
     // extract parameters
     resultSetSerializable.parameters =
         SessionUtil.getCommonParams(rootNode.path("data").path("parameters"));
+    if (resultSetSerializable.parameters.isEmpty()) {
```

Comment:
If no list of common params is returned with resultset, get previous values from session.